conda-osx-64.lock

```diff
@@ -0,0 +1,194 @@
+# Generated by conda-lock.
```

Comment:
If this doesn't work on Mac OS X, why is there still a lock file for OS X?

Reply:
good point, I be removes it

Reply:
I be removed it arrr :pirate_flag: